packages/http-client-java/generator/http-client-generator-clientcore-test/pom.xml

```diff
@@ -64,7 +64,7 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
-      <version>2.18.0</version>
+      <version>2.17.2</version>
```

Comment:
Wonder where 2.18.0 came from.. We had 2.17.2 for quite some time in SDK repo.

Reply:
Likely Hong filled it when adding tests for this module.